process/resultsParser/returnCode.go

```diff
@@ -0,0 +1,36 @@
+package resultsParser
+
+import (
+	"bytes"
+	"encoding/hex"
+)
+
+// ReturnCode is an enum that describes some common smart contract results.
+type ReturnCode string
+
+const (
+	None                   ReturnCode = ""
+	Ok                     ReturnCode = "ok"
+	FunctionNotFound       ReturnCode = "function not found"
+	FunctionWrongSignature ReturnCode = "wrong signature for function"
+	ContractNotFound       ReturnCode = "contract not found"
+	UserError              ReturnCode = "user error"
+	OutOfGas               ReturnCode = "out of gas"
+	AccountCollision       ReturnCode = "account collision"
```

Comment:
Maybe only define the ones that are used within the code?

Reply:
changed.